src/utils/getTokenFromRequest.js

```diff
@@ -0,0 +1,10 @@
+const getTokenFromRequest = (req) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
```

Comment:
Consider adding a check to ensure that the authorization header is in the expected format ('Bearer token'). If the format is incorrect, the function might return an undefined token, which could lead to errors in subsequent processing.